packages/api/src/share_link/service.ts

```diff
@@ -168,9 +168,9 @@ const updateShareLink = async (
 };
 
 const getShareLinks = async (
-	email: string,
 	shareTokens: string[] | undefined,
 	shareLinkIds: string[] | undefined,
+	email?: string,
```

Comment:
I've tended to succumb to the idea that optional parameters (as opposed to required parameters which can be passed `undefined` as an allowed value) are not ideal; in this PR it looks like email *is* always passed, it's just it can be undefined right?  If so, let's consider making the parameter required but allowing `| undefined`.